tools/marvin/marvin/deployDataCenter.py

```diff
@@ -224,6 +225,15 @@ def createClusters(self, clusters, zoneId, podId, vmwareDc=None):
                                                podId,
                                                clusterId)
 
+            threads = []
+            for cluster in clusters:
+                t = threading.Thread(target=createCluster, args=(cluster,))
+                t.start()
+                threads.append(t)
+
+            for t in threads:
+                t.join(1800)
```

Comment:
Instead of hardcoding, better to set it based on host count in the cluster. Also consider that waitForHost() can wait upto 60 secs.

If one t.join() throws what happens to the other threads. Also will __cleanAndExit() behave consistently if there are exceptions in multiple threads.